CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 -   Reformatted demo website to horizontal layout.
 -   Added more documentation for the `--output` flag.
 -   Added zoom functionality to output canvas.
+-   Disabled download buttons when there is no input/output.
```

Comment:
Look carefully at the changelog: this is under the 0.3.2 release (which I made a few days ago), but you should add it to the unreleased section.

Reply:
1. I was planning on making a separate pull request for the bug fix I mentioned on Slack. Would it be better to include it in this one instead? 
2. The Download SVG button was already disabled if there was no svgResult, so instead of modifying the button directly, I made changes so that svgResult is set to null when no image is rendered. Is there a preferable approach I should try instead?

Reply:
Ah I misunderstood the change you made to `svgResult`.

For my latter comment, then, I don't particularly like that now the buttons have two different approaches for their disabling. In the download JSON button, it should be fine to just pass in the `textData` prop and not the `jsonResult` prop. Indeed, in the parent component I'm not convinced the `jsonResult` state is necessary at all; seems like it can just be computed when generating the SVG image.

Reply:
It seems like the `jsonResult` state is used to communicate when the `Draw Diagram` button has been pressed. Computing it while generating the SVG results in the image output updating in real-time while the user makes changes to their input (instead of only updating once `Draw Diagram` is pressed).

Reply:
@yoonieaj ah okay, good research. In this case I think it's fine to keep the `jsonResult` state. But to be clear, you should still make the change I've suggested above: "In the download JSON button, it should be fine to just pass in the `textData` prop and not the `jsonResult` prop."